include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -164,14 +159,13 @@ uninitialized_fill(_ExecutionPolicy&& __exec, _ForwardIterator __first, _Forward
     {
         oneapi::dpl::__internal::__pattern_walk_brick(
             __dispatch_tag, std::forward<_ExecutionPolicy>(__exec), __first, __last,
-            oneapi::dpl::__internal::__brick_fill<decltype(__dispatch_tag), _DecayedExecutionPolicy, _ValueType>{
-                _ValueType(__value)});
+            oneapi::dpl::__internal::__brick_fill<decltype(__dispatch_tag), _ValueType>{_ValueType(__value)});
     }
     else
     {
         oneapi::dpl::__internal::__pattern_walk1(
             __dispatch_tag, std::forward<_ExecutionPolicy>(__exec), __first, __last,
-            oneapi::dpl::__internal::__op_uninitialized_fill<_Tp, _DecayedExecutionPolicy>{__value});
+            oneapi::dpl::__internal::__op_uninitialized_fill<_Tp, std::decay_t<_ExecutionPolicy>>{__value});
```

Comment:
As far as we are changing template parameters of `__op_uninitialized_fill`....
It looks `__op_uninitialized_fill `has only one implementation for the host and device purposes. So, let remove `_ExecutionPolicy` at all.

Reply:
Fixed

Reply:
I don't know from implementation point of view how to do this.
@MikeDvorskiy could you please prepare some separate PR with this fix?